src/pages/docs/service-level-agreement.md

```diff
@@ -0,0 +1,111 @@
+# Business Service Level Agreement
+
+## 1. web3.storage Service Level Agreement ("SLA").
+
+Nitro Data LLC ("Company") commits to provide a level of service for web3.storage Customers demonstrating:
+
+1.1 99.9% Uptime. The Upload Service will upload Customer Content at least 99.9% of the time and the Read Service will serve Customer Content at least 99.9% of the time, subject to Section 4.
+
+1.2 Penalties. If any of the Services fails to meet the above service level, the Customer will receive a Service Discount on their next monthly bill, as further described in Section 6 of this SLA.
+
+1.3 Upload Size Limit. Individual uploads to the Upload Service may have a size limit per API call, as described in the API documentation available at <https://web3.storage/docs/reference/http-api/>.
```

Comment:
We don't have an http API reference

Reply:
@heyjay44 will see we should drop this point

Reply:
1.3 Upload Size Limit. Individual uploads to the Upload Service may encounter issues if they are over 700GB.

Reply:
we changed to 750